ext/rp/timer.h

```diff
@@ -0,0 +1 @@
+// Empty header "timer.h" for tinyusb
```

Comment:
```suggestion
// Empty header "timer.h" for tinyusb

#if defined TUD_OPT_RP2040_USB_DEVICE_UFRAME_FIX && (TUD_OPT_RP2040_USB_DEVICE_UFRAME_FIX == 1)
#error TUD_OPT_RP2040_USB_DEVICE_UFRAME_FIX is not supported in modm currently.
#endif
```
Maybe, just in case someone tries to enable `TUD_OPT_RP2040_USB_DEVICE_UFRAME_FIX`?